app/lib/Emoji.js

```diff
@@ -1,35 +1,55 @@
 import escape from 'escape-html';
-import BUILDKITE_EMOJIS from '../emojis/buildkite';
-import APPLE_EMOJIS from '../emojis/apple';
+import emojiRegex from 'emoji-regex';
```

Comment:
This `emoji-regex` lib was a great find! I went searching for something ages ago but couldn't find it.

Do you know how roughly big the regular expression is? I.e. does it create a unique regex for every emoji that exists? Or is it generic? I couldn't quite figure it out from looking at it's source code.

Either way I don't really care, I was just wondering if it would have a performance impact.